### ISSUE TITLE
Documentation for JSON

### ISSUE THREAD

**Issue by oscarvarto** [dflib/dflib#409]
I've searched https://dflib.org/dflib/docs/1.x/, but I didn't see documentation specific to the super common JSON format.
Am I missing something obvious?

**Comment by andrus** [comment]
Yeah, there's a gap in the docs, but JSON <> DataFrame conversion is supported (albeit still somewhat limited). 

1. Add a dependency:

```xml
<dependency>
    <groupId>org.dflib</groupId>
    <artifactId>dflib-json</artifactId>
</dependency>
```

2. When loading a DataFrame from JSON,  mapping from properties to columns happens either implicitly based on JSON property names or using JSONPath expressions. Some examples are here - https://github.com/dflib/dflib/blob/main/dflib-json/src/test/java/org/dflib/json/JsonLoaderTest.java

3. Saving to JSON currently is very simply - just an array of simple objects, with property names corresponding to columns.

We are planning to improve both loading and saving. So if you have specific use cases, feel free to share.

**Comment by andrus** [comment]
Just published JSON load/save docs at https://dflib.org/dflib/docs/1.x/#json
